grpc/server/src/test/proto/echopackage.proto

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+package helidon.test;
+
+option java_package = "io.helidon.grpc.server.test";
```

Comment:
So this will only test that the `java_package` works as expected, right?

Do we already have the tests that ensure that the `package` is used instead when `java_package` is missing? If not, we should add those as well.

Reply:
No we don't. I can check that out.

Reply:
Added